_component_design/table-variations.md

```diff
@@ -0,0 +1,309 @@
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Table variations
+
+Note to Chris and Bart: This section will include content taken from the Table page.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use the following guidelines when you consider which table variation to construct.
+
+- Use a *multi-line* table when the text in a cell wraps to two or more lines. For example, use a dual-line table when two values are meaningfully related to each other and should be displayed in the same cell.
+
+- Use a *dual-layout* table when you need to display data in two tables, one on the left and one on the right.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you construct table variations.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+### Multi-line table best practices
+
+Consider the following best practices when you construct a dual-line table:
+
+- In lines displaying two independent text rows, emphasize the primary line (on top) by increasing its font weight and adding **bold**.
+- If you use the sort control in the table header, ensure that the sort is based on the primary key.
+- Due to the amount of information that can be displayed in a dual-line table, consider pairing the table with [assisted search]({{site.baseurl}}/components/assisted-search.html). Assisted search provides more robust filtering of table content.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+### Dual-layout table best practices
+
+Consider the following best practices when you construct a dual-layout table:
+
+- xxxx
+- xxxx.
+- xxxx.
+- xxxx.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Multi-line table specifications
+
+Use these specifications when you construct a dual-line table.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Composition
+
+The dual-line table is composed of the following elements:
+
+- **xxxxx (Required/Optional)**: xxxxx.
```

Comment:
primary data point, secondary data point - like Name and ID. Can filter on both primary and secondary, depending on the use case. For example, column for Modified has name as primary, but when sorting happens it is based on the date, which is secondary.

Ask the ticketing team and notification. Ping RED how does sorting work, do you ever sort on the secondary or subordinate information. Slack all of RED.